### DISCUSSION TITLE
How can I change the format of the numbers displayed in my donut chart tooltip?

### DISCUSSION THREAD

**Discussion by LucieSumToZero** [mantinedev/mantine#7122] [Q&A]
How can I control the values displayed in the segment tooltip of my following donut chart ? For instance, to use percentage or scientific format?
Thank you
 ![image](https://github.com/user-attachments/assets/dba46e61-5b92-4cde-8b53-a84178331ba8)

**Comment by rtivital** [comment]
Try using `valueFormatter` prop
